Replace two AAD Graph samples with MS Graph samples

Jackson-Woods opened the pull request:
- Deleted two obsolete C# AAD Graph samples from AAD Graph sample table
- Created new MS Graph sample table and added two comparable C# samples to replace those deleted
- Minor typo corrections

azurecla commented:
Hi __@KBaroni__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

KBaroni commented:
#sign-off

azurecla commented:
Hi __@KBaroni__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

KBaroni commented:
#sign-off

Jackson-Woods committed:
Replace two AAD Graph samples with MS Graph samples

- Deleted two obsolete C# AAD Graph samples from AAD Graph sample table
- Created new MS Graph sample table and added two comparable C# samples to replace those deleted
- Minor typo corrections
